set.mm

```diff
@@ -775593,6 +775742,141 @@ additional condition (deduction form).  See ~ ralbidc for a more
+      $( Two ways of expressing ` A R B ` .  (Contributed by Zhi Wang,
+         18-Sep-2024.) $)
+      fvconstr2 $p |- ( ph -> A R B ) $=
+        ( cop wcel wbr co c0 wne ne0d csn cxp cfv oveqd df-ov eqtrdi neeq1d cdm
+        dmxpss ndmfv necon1ai sseldi syl6bi mpd df-br sylibr ) ABCJZDKZBCDLABCE
+        MZNOZUNAUOFIPAUPUMDGQZRZSZNOZUNAUOUSNAUOBCURMUSAEURBCHTBCURUAUBUCUTURUD
+        ZDUMDUQUEUMVAKUSNUMURUFUGUHUIUJBCDUKUL $.
+    $}
+  $}
+
+
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  ZF Set Theory - add the Axiom of Union
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Ordinal arithmetic
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $( Ordinal 1 is a set.  (Contributed by BJ, 6-Apr-2019.)  (Proof shortened by
+     AV, 1-Jul-2022.)  Remove dependency on ~ ax-10 , ~ ax-11 , ~ ax-12 ,
+     ~ ax-un .  (Revised by Zhi Wang, 19-Sep-2024.) $)
+  1oexw $p |- 1o e. _V $=
+    ( c1o c0 csn cvv df1o2 snex eqeltri ) ABCDEBFG $.
+    $( $j usage '1oexw' avoids 'ax-10' 'ax-11' 'ax-12' 'ax-un'; $)
+
+  $( ` 2o ` is a set.  (Contributed by BJ, 6-Apr-2019.)  Remove dependency on
+     ~ ax-10 , ~ ax-11 , ~ ax-12 , ~ ax-un .  (Proof shortened by Zhi Wang,
+     19-Sep-2024.) $)
+  2oexw $p |- 2o e. _V $=
+    ( c2o c0 c1o cpr cvv df2o3 prex eqeltri ) ABCDEFBCGH $.
+    $( $j usage '2oexw' avoids 'ax-10' 'ax-11' 'ax-12' 'ax-un'; $)
+
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Equinumerosity
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    $d A x $.  $d B x $.  $d X x $.
+    fconst0ci.1 $e |- B e. _V $.
+    fconst0ci.2 $e |- Y = ( ( A X. { B } ) ` X ) $.
+    $( Given a constant function, the function value is either the constant
+       output of the function, or the empty set.  (An artifact of our function
+       value definition.)  (Contributed by Zhi Wang, 18-Sep-2024.) $)
+    fconst0ci $p |- ( Y = (/) \/ Y = B ) $=
+      ( csn cxp cdm wcel c0 wceq wo cfv dmxpss sseli fvconst2 syl syl5eq olcd
+      wn ndmfv orcd pm2.61i ) CABGZHZIZJZDKLZDBLZMUHUJUIUHDCUFNZBFUHCAJUKBLUGAC
+      AUEOPABCEQRSTUHUAZUIUJULDUKKFCUFUBSUCUD $.
+  $}
+
+  ${
+    $d A x $.  $d B x $.  $d X x $.
+    fconstdomi.1 $e |- B e. _V $.
+    $( Given a constant function, the function value is dominated by the
+       constant output of the function.  (An artifact of our function value
+       definition.)  (Contributed by Zhi Wang, 18-Sep-2024.) $)
+    fconstdomi $p |- ( ( A X. { B } ) ` X ) ~<_ B $=
+      ( csn cxp cdm wcel cfv cdom wbr wceq sseli fvconst2 syl cvv domrefg ax-mp
+      dmxpss eqbrtrdi wn c0 ndmfv 0dom pm2.61i ) CABEZFZGZHZCUGIZBJKUIUJBBJUICA
+      HUJBLUHACAUFSMABCDNOBPHBBJKDBPQRTUIUAUJUBBJCUGUCBDUDTUE $.
+  $}
```

Comment:
```suggestion
  ${
    $d A x $.  $d B x $.  $d X x $.
    fvconst0ci.1 $e |- B e. _V $.
    fvconst0ci.2 $e |- Y = ( ( A X. { B } ) ` X ) $.
    $( A constant function's value is either the constant or the empty set.
       (An artifact of our function value definition.)  (Contributed by Zhi
       Wang, 18-Sep-2024.) $)
    fvconst0ci $p |- ( Y = (/) \/ Y = B ) $=
      ( csn cxp cdm wcel c0 wceq wo cfv dmxpss sseli fvconst2 syl syl5eq olcd
      wn ndmfv orcd pm2.61i ) CABGZHZIZJZDKLZDBLZMUHUJUIUHDCUFNZBFUHCAJUKBLUGAC
      AUEOPABCEQRSTUHUAZUIUJULDUKKFCUFUBSUCUD $.
  $}

  ${
    $d A x $.  $d B x $.  $d X x $.
    fvconstdomi.1 $e |- B e. _V $.
    $( A constant function's value is dominated by the constant.  (An artifact
       of our function value definition.)  (Contributed by Zhi Wang,
       18-Sep-2024.) $)
    fvconstdomi $p |- ( ( A X. { B } ) ` X ) ~<_ B $=
      ( csn cxp cdm wcel cfv cdom wbr wceq sseli fvconst2 syl cvv domrefg ax-mp
      dmxpss eqbrtrdi wn c0 ndmfv 0dom pm2.61i ) CABEZFZGZHZCUGIZBJKUIUJBBJUICA
      HUJBLUHACAUFSMABCDNOBPHBBJKDBPQRTUIUAUJUBBJCUGUCBDUDTUE $.
  $}
```

I think fvconst matches with the fvconst* theorems better, though I admit the mnemonic breaks down (constant function's value is sic. function value constant)

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4220/commits/4fb97675bd81c63478d961a49d1ad944f128d680